CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+Changes in [1.0.3](https://github.com/matrix-org/sydent/releases/tag/v1.0.3) (2019-05-03)
+=========================================================================================
+
+Security Fixes
```

Comment:
oops, forgot to add anything here for #150 .

(Note that there are also other, non-security changes since the last release.)